lerna.json

```diff
@@ -2,5 +2,5 @@
   "npmClient": "yarn",
   "useWorkspaces": true,
   "packages": ["packages/*"],
-  "version": "0.4.0"
+  "version": "0.3.0"
```

Comment:
I forgot we were using lerna here, so I'll just let lerna handle version bumping. Reverting here, and right after i'll run a lerna bump